### DISCUSSION TITLE
What happens when a trigger fails?

### DISCUSSION THREAD

**Discussion by derrickburns** [argoproj/argo-events#1475] [Q&A]
The documentation explains that triggers can fail, but does not explain what happens when they do.

To be concrete, here is my question.  Say an event is received by a sensor.  The sensor sends an HTTP message (webhook). The webhook fails.   

What happens to the event?  Is it simply dropped?  Does the sensor retry?  What is the retry policy?

**Comment by whynowy** [comment]
https://github.com/argoproj/argo-events/blob/master/docs/sensors/more-about-sensors-and-triggers.md#trigger-retries
